bikeshare_app/views.py

```diff
@@ -107,6 +107,13 @@ def register():
         # Allow user through if check passed, else log attempt and return home
         return render_template('register.html')
 
+@app.route('/checkin', methods=['GET'])
```

Comment:
Yeah, I just adde the ['GET'] because I thought we would need it when they login. Like an exchange will be made to receive necessary information such as the bike they used, username etc. However, I am not exactly sure how that is going to work in the backend. Should I leave it out for now?